content/docs/user-guide/components/reference/terrain/world.md

```diff
@@ -11,6 +11,14 @@ The **Terrain World** component provides the data required for other terrain sys
 You must add this component to the *Level* entity, the parent of all entities in an **Open 3D Engine (O3DE)** level.
 {{< /important >}}
 
+## Usage 
+
+The **Min Height** and **Max Height** bounds should be set to as small a range as possible for the most accurate height values. For instance, the terrain renderer uses 16 bit height values, which provides 65536 possible distinct heights. If the difference between the min and max height is 1km, that's a resolution of about 1.5cm.
+
+**Height Query Resolution** is a synchronization point for terrain height used by different terrain systems like physics and rendering. For physics, this controls the density of the underlying height field collider. For rendering, this determines the triangle density of the closest level of detail.
+
+**Surface Data Query Resolution (m)** is similar to above, but it controls the resolution of surface data instead of heights. For instance, this affects the density of detail materials for detail material blending.
```

Comment:
Suggestion: It might be worth describing a bit more how this conceptually creates two terrain world grids - one for height, and one for surface data. All height and surface data providers will be queried at this grid resolution, regardless of the resolution of the data in those providers, so the ideal goal is to use data that matches this resolution. For example, if a heightmap with 10 texels/m is used with a terrain that has a 1 m height query resolution, 9/10 of the texels will be ignored. If the heightmap has 1 texel/m with a 1 m height query resolution, then all of the texels will be used.